src/redux/actions/constants.js

```diff
@@ -5,3 +5,4 @@ export const SET_LOADED = 'data/set_loaded'
 export const SET_FILTERS_VISIBILITY = 'filters/set_filters_visibility'
 export const TOGGLE_PROVIDER = 'filters/toggle_provider'
 export const SET_CATEGORY = 'filters/set_category'
+export const SET_FAVOURITES = 'set_favourites'
```

Comment:
`export const SET_FAVOURITES = 'data/set_favourites'` - just to make everything in 1 style